overrides/netlify.toml

```diff
@@ -73,6 +75,7 @@
         https://unpkg.com
         https://script.wiz.gov.sg/widget.css
         https://script-staging.wiz.gov.sg/widget.css
+        https://*.dcube.cloud
```

Comment:
let's add the exact whitelist as given here for all: https://docs.wogaa.sg/web-implementation/setup-guide/information-service-implementation-test/#step-3-content-security-policies-for-staging

Reply:
thanks for catching! https://github.com/opengovsg/isomer-build/pull/48/commits/c3a58654690232c0fd0d959e25a664896d9fc77a